portfolio/src/main/webapp/index.html

```diff
@@ -149,6 +170,7 @@
                 <div style="text-align:center">
                 <span class="dot" onclick="currentSlide(1)"></span>
                 <span class="dot" onclick="currentSlide(2)"></span>
+
```

Comment:
I think you don't need this empty line?  It is like you don't need am empty line before "}" in Java

But it is just my opinion, do let me know if you see some style guide suggesting we should have this kind of empty line.